execution_engine/src/core/engine_state/put_trie.rs

```diff
@@ -0,0 +1,33 @@
+use crate::shared::newtypes::Blake2bHash;
+
+/// Result from inserting a [Trie][storage::trie::Trie] into the trie store.  Contains:
+///
+/// - The trie key the inserted [Trie][storage::trie::Trie] (which is its [Blake2bHash])
+///
+/// - The missing descendant keys of that trie transitively refers to. Found using
```

Comment:
```suggestion
/// - The missing descendant keys which that trie transitively refers to. Found using
```

Reply:
Fixed in https://github.com/CasperLabs/casper-node/pull/816/commits/c8693d3a44055a74062b62864eafbfd2eea7190e